keps/prod-readiness/sig-node/4540.yaml

```diff
@@ -1,4 +1,4 @@
 kep-number: 4540
-alpha:
```

Comment:
You leave alpha as is and just add a beta section.

Reply:
ok, got you.